src/Facebook/InstantArticles/Elements/H2.php

```diff
@@ -65,16 +65,28 @@ public function withTextAlignment($text_alignment)
     }
 
     /**
-    * The Text position that will be used.
```

Comment:
@simonengelhardt 

Unrelated to this PR, but there are a lot of docblocks which aren't indented correctly.